package VideoFramework;
option optimize_for = SPEED;

message FlowPoint {
  optional float x = 1;
  optional float y = 2;
}

// Records initial position and matching points in prev. frames.
message FlowMatches {
  repeated FlowPoint point = 1;
}

// Stores all feature locations and matches.
message FlowFrame {
  repeated FlowMatches matches = 1;
}
